src/logger.js

```diff
@@ -1,10 +1,19 @@
-import { isDebug } from './utils';
+import { TracerGlobals } from './globals';
 
 const LOG_PREFIX = '#LUMIGO#';
 const WARN_CLIENT_PREFIX = 'Lumigo Warning';
 
+export const isDebug = () => {
```

Comment:
Moved this function from utils.js because now utils.js uses logger.js and I wanted to avoid circular import.
Also, I think it makes sense to get the log level from the logger.

Reply:
🚀